app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/adapters/recycleradapters/FeedbackAdapter.kt

```diff
@@ -26,6 +30,7 @@ class FeedbackAdapter(
         RecyclerView.Adapter<FeedbackViewHolder>(), FeedbackViewHolder.ClickListener {
 
     private val clickListener: FeedbackViewHolder.ClickListener = this
+    var arrangedFeedbackList: ArrayList<Feedback> = ArrayList()
```

Comment:
Where are you reassigning the variable? From which external class are you accessing the variable?

Seems I have to repeat the same review again and again. I will not review again unless you learn from past reviews and stop making same mistakes again and again

Reply:
> Where are you reassigning the variable? From which external class are you accessing the variable?
> 
> Seems I have to repeat the same review again and again. I will not review again unless you learn from past reviews and stop making same mistakes again and again

Sorry for the inconvenience caused. I will definitely take care of now and not give any other chance for such mistakes.
 I have updated it. Also, I have changed it to getStringSet, else the app is crashing. During the successful login, the email id is saved using StringSet. So we have to use stringSet to retrieve the data.